pkg/monitoring/daemonsets_test.go

```diff
@@ -14,24 +13,16 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func TestCreateFilebeatDaemonSet(t *testing.T) {
+func TestCreateSystemDaemonSet(t *testing.T) {
 	assert := assert.New(t)
 
 	const clusterName = "cluster1"
 	const uri = "vzURI"
 	nameMap := map[string]bool{
-		constants.FilebeatName:     true,
-		constants.JournalbeatName:  true,
 		constants.NodeExporterName: true}
-	dsets := SystemDaemonSets(clusterName, uri, clusterInfoContainerd)
+	dsets := SystemDaemonSets(clusterName, uri, ClusterInfo{ContainerRuntime: "containerd://1.4.0"})
```

Comment:
this seems very brittle

Reply:
It was defined as a var in the removed configmaps_test.go:
-var clusterInfoDocker = ClusterInfo{ContainerRuntime: "docker://19.3.11"}
-var clusterInfoContainerd = ClusterInfo{ContainerRuntime: "containerd://1.4.0"}